core/templates/domain/voiceover/entity-voiceovers.model.ts

```diff
@@ -75,28 +98,72 @@ export class EntityVoiceovers {
       let voiceoverTypeToVoiceovers =
         entityVoiceoversBackendDict.voiceovers_mapping[contentId];
 
-      contentIdToVoiceoversMapping[contentId] = {
-        manual: Voiceover.createFromBackendDict(
+      let manualVoiceovers!: Voiceover;
+      let automaticVoiceovers!: Voiceover;
+
+      if (voiceoverTypeToVoiceovers.manual) {
+        manualVoiceovers = Voiceover.createFromBackendDict(
           voiceoverTypeToVoiceovers.manual
-        ),
+        );
+      }
+
+      if (voiceoverTypeToVoiceovers.auto) {
+        automaticVoiceovers = Voiceover.createFromBackendDict(
+          voiceoverTypeToVoiceovers.auto
+        );
+      }
+
+      contentIdToVoiceoversMapping[contentId] = {
+        manual: manualVoiceovers,
+        auto: automaticVoiceovers,
       };
     }
+
+    let contentIdToVoiceoversAudioOffsetsMsecs: ContentIdToVoiceoversAudioOffsetsMsecs =
+      {};
+
+    for (let contentId in entityVoiceoversBackendDict.automated_voiceovers_audio_offsets_msecs) {
+      let audioOffsets =
+        entityVoiceoversBackendDict.automated_voiceovers_audio_offsets_msecs[
+          contentId
+        ];
+      contentIdToVoiceoversAudioOffsetsMsecs[contentId] = audioOffsets.map(
+        audioOffset => {
+          return {
+            token: audioOffset.token,
+            audioOffsetMsecs: audioOffset.audio_offset_msecs,
+          };
+        }
+      );
+    }
+
     return new EntityVoiceovers(
       entityVoiceoversBackendDict.entity_id,
       entityVoiceoversBackendDict.entity_type,
       entityVoiceoversBackendDict.entity_version,
       entityVoiceoversBackendDict.language_accent_code,
-      contentIdToVoiceoversMapping
+      contentIdToVoiceoversMapping,
+      contentIdToVoiceoversAudioOffsetsMsecs
     );
   }
 
   getManualVoiceover(contentId: string): Voiceover | undefined {
     let voiceoverTypeToVoiceovers = this.voiceoversMapping[contentId];
 
-    if (voiceoverTypeToVoiceovers) {
+    if (voiceoverTypeToVoiceovers && voiceoverTypeToVoiceovers.manual) {
       return voiceoverTypeToVoiceovers.manual;
     } else {
       return undefined;
     }
   }
+
+  getAutomaticVoiceover(contentId: string): Voiceover | undefined {
+    let voiceoverTypeToVoiceovers = this.voiceoversMapping[contentId];
+
+    if (voiceoverTypeToVoiceovers && voiceoverTypeToVoiceovers.auto) {
+      return voiceoverTypeToVoiceovers.auto;
+    } else {
+      return undefined;
+    }
```

Comment:
```suggestion
      return this.voiceoversMapping[contentId]?.auto;
```

Ditto to obtain manual voiceover.

Reply:
Done